README.md

```diff
@@ -8,7 +8,7 @@ I made this quick package to limit the amout of times I was implementing this fu
 ## Install 
 
 ```bash
-$ yarn add nestjs-typeorm-pagination
+$ yarn add nestjs-typeorm-paginate
```

Comment:
can we then rename the github repo to `nestjs-typeorm-paginate` as well? in order to have the same names? just for compliance and stuff? :laughing:

Reply:
Yea sure :)

Reply:
And now.. turn off your smartphone ;)